editor/components/default-block-appender/style.scss

```diff
@@ -1,7 +1,10 @@
 $empty-paragraph-height: $text-editor-font-size * 4;
 
 .editor-default-block-appender {
-	.editor-default-block-appender__content {
+	// Componsate for outline being applied outside the appender
```

Comment:
Comment still applicable?

Reply:
Nope. Good catch. Also refactored this entire SCSS file a bit. See next comment.